docs/src/specs/crypto.md

```diff
@@ -5,12 +5,10 @@ Anoma currently supports Ed25519 signatures with more to be added:
 - [`Secp256k1`](https://github.com/anoma/anoma/issues/162)
 - [`Sr25519`](https://github.com/anoma/anoma/issues/646)
 
-Please note that the Anoma's crypto public API and encoding is currently undergoing some breaking changes with <https://github.com/anoma/anoma/issues/225>.
-
 ## Public keys
 
-A public key is a [Borsh encoded `PublicKey`](encoding.md#publickey). For the Ed25519 scheme, this is 32 bytes of Ed25519 public key, prefixed with `32` in little endian encoding (`[32, 0, 0, 0]` in raw bytes or `20000000` in hex). (TODO this will change with <https://github.com/anoma/anoma/issues/225>)
+A public key is a [Borsh encoded `PublicKey`](encoding.md#publickey). For the Ed25519 scheme, this is 32 bytes of Ed25519 public key, prefixed with `32` in little endian encoding (`[32, 0, 0, 0]` in raw bytes or `20000000` in hex).
```

Comment:
I'm unsure of the context here for where we're using Borsh-encoded public keys and signatures, but I'm guessing it's when we store them in blockchain state?

Reply:
I'd be tempted to remove references to encoding here, that could be covered in the next section (https://dev.anoma.net/master/specs/encoding.html)

Reply:
yeah, I think this is outdated now. We're using the `key::common` schema with enum cases for ed25519 and upcoming secp256k1